db/migrate/20170420160820_copy_description_values_to_description_string.rb

```diff
@@ -0,0 +1,12 @@
+class CopyDescriptionValuesToDescriptionString < ActiveRecord::Migration[5.0]
```

Comment:
I think we'll want to disable ddl transaction on this so that we don't lock up all editions while this runs